src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,64 @@
-export const Tabs = () => {};
+import { FC, useState } from 'react';
+import classNames from 'classnames';
+import { Tab } from './types/interface';
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void;
+}
+
+export const Tabs: FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  let currentTab = tabs.find(({ id }) => id === selectedTabId);
+
+  if (!currentTab) {
+    const [tab] = tabs;
+
+    currentTab = tab;
+  }
+
+  const [text, setText] = useState(currentTab.content);
+  const choseTab = (tab: Tab) => {
```

Comment:
The `choseTab` function should not be necessary since the `Tabs` component should be stateless. Remove the `choseTab` function and use the `onTabSelected` callback directly in the `onClick` event.